src/DssDirectDepositHub.sol

```diff
@@ -113,6 +144,14 @@ contract DssDirectDepositHub {
     event Quit(bytes32 indexed ilk, address indexed usr);
     event Exit(bytes32 indexed ilk, address indexed usr, uint256 amt);
 
+    /**
+        @dev sets msg.sender as authed
+        Allows joining/exiting Dai by:
+           - giving infinite approval for Dai on the DaiJoin contract
+           - hoping the DaiJoin in the Vat
```

Comment:
These seem a bit too detailed to me.

Reply:
Not critical but I see the detailed explanation is still there (just not in bullets).
Let me know if you prefer to leave as is.